bundles/org.openhab.binding.somfymylink/src/main/java/org/openhab/binding/somfymylink/internal/model/SomfyMyLinkCommandSceneList.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.somfymylink.internal.model;
+
+/**
+ * @author Chris Johnson - Initial contribution
+ */
+public class SomfyMyLinkCommandSceneList extends SomfyMyLinkCommandShadeBase {
```

Comment:
To deal with the `@NonNullByDefault` warnings you can do one of three things:
1. Add the `@NonNullByDefault` to the class.
2. Make sure that your class name ends in DTO
3. Make sure that your class is in a package or subpackage called `dto`.

Please do one of the three.
If it is easy to do I would suggest doing the first one, otherwise do the third.

Please do the same for the other command classes.

Reply:
ok. got it. thx. done.